client/http/api.go

```diff
@@ -38,6 +38,9 @@ const (
 	store                     = "/pd/api/v1/store"
 	Stores                    = "/pd/api/v1/stores"
 	StatsRegion               = "/pd/api/v1/stats/region"
+	MembersPrefix             = "/pd/api/v1/members"
+	LeaderPrefix              = "/pd/api/v1/leader"
+	TransferLeader            = "/pd/api/v1/leader/transfer"
```

Comment:
If these two consts will always be used with the exported functions below, it's better to keep them unexported.

```suggestion
	leaderPrefix              = "/pd/api/v1/leader"
	transferLeader          = "/pd/api/v1/leader/transfer"
```